server/src/test/java/io/spine/core/EventsShould.java

```diff
@@ -48,6 +49,7 @@
 import static io.spine.core.Events.getTimestamp;
 import static io.spine.core.Events.nothing;
 import static io.spine.core.Events.sort;
+import static io.spine.core.given.EventsTestEnv.TEST_TENANT_NAME;
```

Comment:
Please have a static method instead of referencing a constant.

I also suspect that it should be `EventsTestEnv.tenantId()` and it should return `TenantId`. Note the absence of "test" and a non-`String` return value.